airflow/jobs/scheduler_job.py

```diff
@@ -389,34 +389,34 @@ def manage_slas(self, dag, session=None):
         Finding all tasks that have SLAs defined, and sending alert emails
         where needed. New SLA misses are also recorded in the database.
 
-        Where assuming that the scheduler runs often, so we only check for
+        We are assuming that the scheduler runs often, so we only check for
         tasks that should have succeeded in the past hour.
         """
-        if not any([isinstance(ti.sla, timedelta) for ti in dag.tasks]):
+        if not any(isinstance(ti.sla, timedelta) for ti in dag.tasks):
             self.log.info("Skipping SLA check for %s because no tasks in DAG have SLAs", dag)
             return
 
+        # Find the last executed tasks
         TI = models.TaskInstance
         sq = (
```

Comment:
Doing such refactor it would be a good idea to change this `sq` name to something more pylint compatible. Maybe then we will be able to remove it from [TODO list](https://github.com/apache/airflow/blob/master/scripts/ci/pylint_todo.txt)?

Reply:
Thanks, didn't know about this one. Will change.

Reply:
hm seems that file has several issues. I would prefer to keep the scope of this ticket on the specific function.